plugins/snippets/luasnip/default.nix

```diff
@@ -13,6 +13,20 @@ in {
 
     package = helpers.mkPackageOption "luasnip" pkgs.vimPlugins.luasnip;
 
+    extraConfig = mkOption {
+      type = types.attrs;
```

Comment:
Please use `types.attrsOf types.anything`, it is better handled in recursive merges of values.

Reply:
Updated